code/package.json

```diff
@@ -72,7 +72,7 @@
     "@vscode/iconv-lite-umd": "0.7.0",
     "@vscode/policy-watcher": "^1.1.4",
     "@vscode/proxy-agent": "^0.18.1",
-    "@vscode/ripgrep": "^1.15.6",
+    "@vscode/ripgrep": "^1.15.7",
```

Comment:
as far as I understand the upstream is still uses `1.15.6`version: https://github.com/microsoft/vscode/blob/047cf79f5d10df18fcf03dbdffe84c107d5c8cbd/package.json#L75

I'm afraid the current change will be overwritten by the upstream changes at the next synchronization Che-Code with the VS Code - so in few hours...

Reply:
to avoid it the corresponding part should be provided to the `.rebase` folder

Reply:
the last 2 merges / rebases from upsteam didn't touch the code/yarn.lock or code/remote/yarn.lock:

https://github.com/che-incubator/che-code/commit/23ace8433f67a10f754c3cf5242668e02689cf99

https://github.com/che-incubator/che-code/commit/61ab6eeeb4bdfdf037ab63afe17087935694b7d5#diff-097ceb3f628ecb473c665dbf54d6571f0b282e6a6f91486bc0c06b74960c243d

But this should be rebased against 7.77.x and merged there, where rebasing from upstream doesn't happen. Correct @RomanNikitenko  ?

Reply:
> But this should be rebased against 7.77.x and merged there, where rebasing from upstream doesn't happen.

correct!
I think we don't need this change for the main branch as VS Code is going to update the version this week.